src/BuiltInTools/dotnet-format/dotnet-format.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
ALl source code of dotnet-format.csproj are put under `src/BuiltIntools/dotnet-format`